common/utils.go

```diff
@@ -92,6 +92,39 @@ func GetFilesByExt(inputPath string, exts []string) ([]string, error) {
 	return files, nil
 }
 
+// GetFilesByExtInCurrDir returns the files present in current directory which have one of the specified extensions
+func GetFilesByExtInCurrDir(inputPath string, exts []string) ([]string, error) {
+	var files []string
+	if info, err := os.Stat(inputPath); os.IsNotExist(err) {
+		logrus.Warnf("Error in walking through files due to : %q", err)
+		return nil, err
+	} else if !info.IsDir() {
+		logrus.Warnf("The path %q is not a directory.", inputPath)
```

Comment:
Should we not just check the extension of the file here and return? If not, won't readdir command fail here?

Reply:
Yes, we should check the extension of the file here. Thanks for detecting the corner case. 
Done.